satpy/composites/__init__.py

```diff
@@ -1677,11 +1677,18 @@ def __call__(self, *args, **kwargs):
 
 
 class BackgroundCompositor(GenericCompositor):
-    """A compositor that overlays one composite on top of another."""
+    """A compositor that overlays one composite on top of another.
 
-    def __call__(self, projectables, *args, **kwargs):
+    Args:
+        bg_fill_in (bool): True means the compositor will fill the area where
+                           foreground is Nan with background.
+                           False means it will just leave the area blank.
```

Comment:
"blank" -> "transparent"?

Reply:
> "blank" -> "transparent"?

Ok. I'm still working on this thing.

Reply:
Understood. I'm just trying to understand in case there are other ways of doing this that already exist.